crates/biome_cli/src/execute/traverse.rs

```diff
@@ -568,9 +572,11 @@ fn process_messages(options: ProcessMessagesOptions) {
     }
 
     for diagnostic in diagnostics_to_print {
-        console.error(markup! {
-            {if verbose { PrintDiagnostic::verbose(&diagnostic) } else { PrintDiagnostic::simple(&diagnostic) }}
-        });
+        if diagnostic.severity() == *diagnostic_level {
```

Comment:
The operator should be `>=`. The idea is that if `diagnostic_level` is `Severity::Information`, we should messages that are `Information` **and higher**.

Reply:
Oh yes, thanks, this actually took care of the majority if the test errors. Coming from TS it never even occurred to me I can use the `>=` operator, what an awesome language really.

Reply:
Indeed! We took that for free by using the traits `Ord` and `PartialOrd` :)